test/functional/test_command.py

```diff
@@ -183,8 +183,12 @@ def test_command_create(scenario_to_test, with_scenario, scenario_name):
         'driver_name',
         'scenario_name',
     ])
-def test_command_dependency_ansible_galaxy(scenario_to_test, with_scenario,
-                                           scenario_name):
+def test_command_dependency_ansible_galaxy(request, scenario_to_test,
+                                           with_scenario, scenario_name):
+    # FIXME(decentral1se): skipped due to failures on network access
+    if request.getfixturevalue('driver_name') != 'docker':
```

Comment:
I'm pretty sure that you could've just added `driver_name` as an argument of the test function...

Reply:
Nah, didn't work ...